src/test/resources/web.xml

```diff
@@ -9,12 +9,13 @@
     <servlet-name>encoding</servlet-name>
     <url-pattern>/encoding</url-pattern>
   </servlet-mapping>
-  <servlet>
-    <servlet-name>multiple-set-cookie</servlet-name>
-    <servlet-class>com.github.jscookie.javacookie.test.integration.write.MultipleSetCookieServlet</servlet-class>
-  </servlet>
-  <servlet-mapping>
-    <servlet-name>multiple-set-cookie</servlet-name>
-    <url-pattern>/multiple-set-cookie</url-pattern>
-  </servlet-mapping>
+  <!-- Test is missing -->
```

Comment:
Really, was this running a test that didn't exist on `master`? If that's the case can we just remove it?

Reply:
I couldn't find it 😉 I'll gladly remove it, but I kept it for reference for your (or any other contributors) input on that matter.